src/lib/apis/most-popular.api.ts

```
@@ -0,0 +1,43 @@
+        'Content-Type': 'application/json',
+      },
+    });
+
+    if (!payload.ok){
+      throw new Error(`Failed to fetch occasions: ${payload.status}`);
+    } 
+    const data = await payload.json();
+    return data;
+
+  } catch (error) {
+    console.error("Error fetching occasion:", error);
```

ESLint

[error] Unexpected console statement.

(no-console)